src/test/java/pages/AllSongsPage.java

```diff
@@ -0,0 +1,46 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.Test;
+
+public class AllSongsPage extends BasePage{
+
+
+    By allSongsElement = By.xpath("//*[@id='sidebar']/section[1]/ul/li[3]/a");
+    By firstSong = By.xpath("//*[@id='songsWrapper']/div/div/div[1]/table/tr[1]/td[2]");
+    By contextMenuAddTo = By.xpath("//*[@id='app']/nav/ul/li[4]");
+
+    By contextMenuAddToPlaylist = By.xpath("//*[@id='app']/nav/ul/li[4]/ul/li[7]");
```

Comment:
If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```